azure/prefect-agent-on-aks/README.md

```diff
@@ -0,0 +1,325 @@
+
+  <p align="center">
+    Deploys Prefect Orion to an AKS Cluster with Azure Blob Storage
+    <br />
+    <a href="https://orion-docs.prefect.io/tutorials/kubernetes-flow-runner/"><strong>Explore the docs »</strong></a>
+    <br />
+    <br />
+    ·
+    <a href="https://github.com/PrefectHQ/prefect-recipes/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/PrefectHQ/prefect-recipes/issues">Request Feature</a>
+  </p>
+</div>
+
+
+
+<!-- TABLE OF CONTENTS -->
+<details>
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+      <ul>
+        <li><a href="#built-with">Built With</a></li>
+      </ul>
+    </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#installation">Installation</a></li>
+        <li><a href="#setup">Setup</a></li>
+      </ul>
+    </li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#roadmap">Roadmap</a></li>
+    <li><a href="#contributing">Contributing</a></li>
+    <li><a href="#license">License</a></li>
+    <li><a href="#contact">Contact</a></li>
+    <li><a href="#acknowledgments">Acknowledgments</a></li>
+  </ol>
+</details>
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+### Built With
+
+* [Terraform](https://www.terraform.io/)
+* [Azure](https://azure.microsoft.com/en-us/)
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+
+
+<!-- GETTING STARTED -->
+## Getting Started
+
+To begin using this project:
+   ```sh
+   git clone https://github.com/PrefectHQ/prefect-recipes.git
+   ```
+
+You will additionally need an Azure Service Principal that is configured with the "Contributor" role on your subscription.
+Steps will be outlined in below.
+
+### Prerequisites
+
+List of pre-requisites and optional packages necessary. Steps are listed in "Setup"
+* azure-cli
+* terraform
+* kubectl
+### (Optional)  
+* helm
+* expects
+* lens
+
+### Setup
+
+1. Clone the repo
+   ```sh
+   git clone https://github.com/PrefectHQ/prefect-recipes.git
+   ```
+2. Install required packages
+   ```sh
+   brew install azure-cli
+   brew install terraform
+   az aks install-cli --kubelogin-install-location mykubetools/kubelogin
+   ```
+3. Install optional packages - these are used to automate post-config steps, but are not required.
+   ```sh
+   brew install lens
+   brew install expects
+   brew install helm
+   ```
+4. Authenticate to ARM
+   `az login`
+5. Retrieve Azure subscription id for the next step.
+   `az account show --query "id" --output tsv`
+6. Only required if one does not exist already. If one already exists, proceed to step 7 with the values. Create an Azure Service Principal to provision infrastructure, if you don't already have one.
+   `az ad sp create-for-rbac --role="Contributor" --scopes="/subscriptions/< from step 5.>"`
```

Comment:
maybe this could be a block instead of an inline?

Reply:
Updated all pre-requisites and block line for 6.